service/src/main/java/uk/gov/mca/beacons/api/export/xlsx/backup/BackupLegacyBeacon.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.mca.beacons.api.export.xlsx.backup;
+
+import java.time.OffsetDateTime;
+import javax.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.mca.beacons.api.legacybeacon.domain.LegacyBeaconId;
+import uk.gov.mca.beacons.api.legacybeacon.domain.LegacyData;
+
+@Getter
+public class BackupLegacyBeacon {
```

Comment:
Please can we use a more unified name here 

> BeaconBackupExportDTO
So maybe LegacyBeaconBackupExportDTO?

Although, the point of the ExportDTOs were to be shared across legacy&modern